packages/runtime/datastore/src/channelStorageService.ts

```diff
@@ -50,7 +50,16 @@ export class ChannelStorageService implements IChannelStorageService {
             ? (await this.extraBlobs).get(id)
             : undefined;
 
-        return blob ?? this.storage.read(id);
+        return blob ?? this.storage.readString(id);
+    }
+
+    public async readString(path: string): Promise<string> {
```

Comment:
Your intuition of needed to add new method here is correct, but if you look at IChannelStorageService interface - it's missing this method, and thus nobody will be able to call it. So it needs to be added there.